### ISSUE TITLE
Library example doesn't work

### ISSUE THREAD

**Issue by zachsa** [blacha/cogeotiff#941]
Hi,

I'm looking for way to read COGs directly from a browser - I wanted to test this library, but the example doesn't seem to work:

```
import { CogSourceUrl } from '@chunkd/source-url';
```

Does that import exist?

**Comment by xycarto** [comment]
I am having a similar issue.

Install with node v12.22.22

```
nvm install v12
npm install @cogeotiff/cli
```

This seems to install 

@chunkd
@cogeotiff

Running

```
import { CogSourceUrl } from '@chunkd/source-url';
```
Produces the error:

```
Loading module from “http://localhost:5173/main.js” was blocked because of a disallowed MIME type (“”).
```
Suggesting the module no longer exists.

Looking into the `node_modules` confirms this and instead I get a new collection of dirs inside @chunkd

```
core
chunkd/fs
source-aws
source-file
source-google-cloud
source-http
```

I guess the question is, which of these is used to read a cogtiff from an s3 end point.  I am guessing it'd be `source-aws`, but I am not sure of the function name.

**Comment by blacha** [comment]
Firstly apologies for missing this issue and taking soo long to respond.

The documentation and examples are lacking for this repository, in this specific case you would need to `yarn add @cogeotiff/core @chunkd/source-url` to your source code and then that example should work.

I have pushed a more complete example with #966 which loads a tile from a COG and renders it into a `<img />` element on the page.

About a year ago I was playing with dynamically layering COGs into a tiling webmap (Leaflet) I have pushed the source code for these experiments to https://github.com/blacha/cogeotiff-web and have deployed the latest source to https://blayne.chard.com/cogeotiff-web/index.html the example COG is stored in New Zealand so your rendering latency may vary quite a bit depending on how far from New Zealand you are. (its a 850MB COG)
